packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -575,13 +575,41 @@ const DropDownEditor = TextBox.inherit({
             'contentReady': this._contentReadyHandler.bind(this)
         });
 
+        eventsEngine.on(this._popup._$content, addNamespace('keydown', this.NAME), (e) => this._popupKeyHandler(e));
```

Comment:
We have a public getter for content
```suggestion
        eventsEngine.on($(this._popup.content()), addNamespace('keydown', this.NAME), (e) => this._popupKeyHandler(e));
```

Reply:
`this._popup._$content` - is `dx-overlay-content`, public getter returns `dx-popup-content` which doesn't include title, toolbar etc

Reply:
Oh, sure. In this case we can use `$overlayContent` method (or similar). I believe that direct getting of a private prop is worse.

Reply:
fair point. done